ci-operator/config/openshift/project-request-limit/openshift-project-request-limit-master.yaml

```diff
@@ -0,0 +1,47 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: "4.5"
+    namespace: ocp
+    tag: base
+binary_build_commands: make build --warn-undefined-variables
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+images:
+- dockerfile_path: images/ci/Dockerfile
+  from: base
+  to: project-request-limit
+promotion:
+  name: "4.5"
+  namespace: ocp
+resources:
+  '*':
+    limits: {}
+    requests:
+      cpu: "3"
+      memory: 1Gi
+tag_specification:
+  cluster: https://api.ci.openshift.org
+  name: "4.5"
+  namespace: ocp
+test_binary_build_commands: make build GO_BUILD_FLAGS:='-race' --warn-undefined-variables
+tests:
+- artifact_dir: /tmp/artifacts
+  as: unit
+  commands: |
+    mkdir -p /tmp/artifacts
+    make test-unit JUNITFILE=/tmp/artifacts/junit_report.xml --warn-undefined-variables
+  container:
+    from: test-bin
+- as: verify
+  commands: make verify
+  container:
+    from: src
+- as: e2e-aws
+  commands: TEST_SUITE=openshift/conformance/parallel run-tests
```

Comment:
missing serial and upgrade test

Reply:
@tnozicka  I don't think the operand needs it unless it is a convention to add `serial and upgrade` across the board for all repos. 
May be the operator repo will have a use for these, although I doubt it. This operator is not a cvo enabled operator and is independent of OpenShift release. When the cluster upgrades, the operator (if installed) will be left untouched.